test/mjsunit/wasm/outer-memory.js

```diff
@@ -0,0 +1,142 @@
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Can we make this test part of the other memory test? The reason is that it was quite painful to update each test after renumbering opcodes (which will happen again in a little while).